src/main/redux/sagas/keyboard.ts

```diff
@@ -35,32 +41,65 @@ function* setShortcuts(action: keyboardActions.setShortcuts.TAction) {
         call(debugShortcuts),
     ]);
 }
-function* reloadShortcuts(action: keyboardActions.reloadShortcuts.TAction) {
-    const loadShortcutsFromJson = async () => {
-        if (action.payload.defaults) {
-            keyboardShortcuts.loadDefaults();
-        }
-        const okay = action.payload.defaults ? true : keyboardShortcuts.loadUser();
+// function* reloadShortcuts(action: keyboardActions.reloadShortcuts.TAction) {
+//     const loadShortcutsFromJson = async () => {
+//         if (action.payload.defaults) {
+//             keyboardShortcuts.loadDefaults();
+//         }
+//         const okay = action.payload.defaults ? true : keyboardShortcuts.loadUser();
 
-        debug(`Keyboard shortcuts reload JSON (defaults: ${action.payload.defaults}) => ${okay}`);
-        if (okay) {
-            put(keyboardActions.setShortcuts.build(keyboardShortcuts.getAll()));
-        }
-    };
+//         debug(`Keyboard shortcuts reload JSON (defaults: ${action.payload.defaults}) => ${okay}`);
+//         if (okay) {
+//             yield put(keyboardActions.setShortcuts.build(keyboardShortcuts.getAll()));
+//         }
+//     };
 
-    yield all([
-        call(loadShortcutsFromJson),
-    ]);
-}
+//     yield all([
+//         call(loadShortcutsFromJson),
+//     ]);
+// }
 
 function* keyboardSetWatcher() {
     yield takeEvery(keyboardActions.setShortcuts.build, setShortcuts);
 }
 function* keyboardShowWatcher() {
     yield takeEvery(keyboardActions.showShortcuts.build, showShortcuts);
 }
+// function* keyboardReloadWatcher() {
+//     yield takeEvery(keyboardActions.reloadShortcuts.build, reloadShortcuts);
+// }
 function* keyboardReloadWatcher() {
-    yield takeEvery(keyboardActions.reloadShortcuts.build, reloadShortcuts);
+    while (true) {
+        const action = yield* takeTyped(keyboardActions.reloadShortcuts.build);
```

Comment:
but why ??

you should may be surround `yield takeEvery` with `yield all([takeEvery])` ,
perhaps it's the raison.

Reply:
There are several issues of this type in this PR. I propose we fix them in a second pass (subsequent PR).